test/CMakeLists.txt

```diff
@@ -42,10 +42,9 @@ configure_lit_site_cfg(
 
 add_lit_testsuite(check-llvm-dialects-lit "Running the llvm-dialects regression tests"
   ${CMAKE_CURRENT_BINARY_DIR}
-  ${exclude_from_check_all}
-  DEPENDS ${LLVM_DIALECTS_TEST_DEPENDS}
+  ${EXCLUDE_FROM_CHECK_ALL}
+  DEPENDS llvm-dialects-test-depends
 )
-set_target_properties(check-llvm-dialects-lit PROPERTIES FOLDER "Tests")
```

Comment:
Why did you remove this? AFAIK it is relevant for Visual Studio.

Reply:
It is already set on llvm-dialects-test-depends (not sure if it works though, going to double-check)